buildscripts/build_qrencode.sh

```diff
@@ -14,13 +14,12 @@ parse_arch --dep "qrencode" --supported "win32 win64" "$@"
 
 "${SCRIPT_DIR}/download/download_qrencode.sh"
 
-CFLAGS="-O2 -g0" ./configure "${HOST_OPTION}" \
-                            --prefix="${DEP_PREFIX}" \
-                            --enable-shared \
-                            --disable-static \
-                            --disable-sdltest \
-                            --without-tools \
-                            --without-debug
+cmake . \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_INSTALL_PREFIX="${DEP_PREFIX}" \
+    "${CMAKE_TOOLCHAIN_FILE}" \
```

Comment:
Tricky that this includes `-DCMAKE_TOOLCHAIN_FILE`. :D

Reply:
Yeah.. not super clean, but need a way to remove the arguments where macOS doesn't need them.